src/OpenTelemetry/Internal/OpenTelemetrySdkEventSource.cs

```diff
@@ -162,6 +172,36 @@ public void MetricObserverCallbackError(string metricName, string exception)
             this.WriteEvent(16, metricName, exception);
         }
 
+        [Event(17, Message = "Batcher finished collection with '{0}' metrics.", Level = EventLevel.Informational)]
+        public void BatcherCollectionCompleted(int count)
+        {
+            this.WriteEvent(17, count);
+        }
+
+        [Event(18, Message = "Collection completed in '{0}' msecs.", Level = EventLevel.Informational)]
+        public void CollectionCompleted(long msec)
+        {
+            this.WriteEvent(18, msec);
+        }
+
+        [Event(19, Message = "Exception occured in Metric Controller. Exception: '{0}'", Level = EventLevel.Warning)]
+        public void MetricObserverException(string exception)
+        {
+            this.WriteEvent(19, exception);
+        }
+
+        [Event(20, Message = "Meter Collect Invoked for Meter: '{0}'", Level = EventLevel.Verbose)]
+        public void MeterCollectInvoked(string meterName)
+        {
+            this.WriteEvent(20, meterName);
+        }
+
+        [Event(21, Message = "Metric Exporter returned error '{0}'.", Level = EventLevel.Warning)]
+        public void MetricExporterErrorResult(MetricExporter.ExportResult exportResult)
```

Comment:
hm, does it work when the argument is an enum and the `Write` is called with the `string`?

Reply:
Yes it was printing some strings. I'll change it to int, and cast the Enum to int.
(Will need similar change in TraceExporter, will make it as a separate PR)